tests/analyzer_docker.test.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import subprocess
+from pathlib import Path
+from sys import argv
+from json import loads
+
+class BuildErr(BaseException):
+    def __init__(self):
+        pass
+
+
+success_msg = '{"success": false, "input": {}, "errorMessage": "Input file doesnt exist"}'
+
+config_required_msg="""  File "/usr/local/lib/python3.8/site-packages/cortexutils/worker.py", line 31, in __init__
```

Comment:
This error message is produced by executing an Analyzer without piping in a relevant job config. Despite it being a stack trace, its presence indicates that the Python interpreter did not detect any syntax errors or missing libraries-- a success in terms of packaging for deployment.